shopelectro/tests/tests_selenium.py

```diff
@@ -271,6 +271,23 @@ def test_add_to_cart(self):
             'js-cart-is-empty')
         self.assertFalse(cart_is_empty.is_displayed())
 
+    def test_add_to_cart_after_load_more(self):
+        """
+        After click at button load more we can add loaded item to cart
```

Comment:
We are able to add loaded product to Cart after `Load more` button click on Category page.